Frontend/ui-common/src/Common/Styles/Colors.purs

```diff
@@ -106,3 +98,40 @@ floralWhite = "#FFFAEE"
 
 orange900 :: Color
 orange900 = "#FF8533"
+
+pickledBlue :: Color
+pickledBlue = "#29334A"
+
+linen :: Color 
+linen = "#FEF1E5" 
+
+pigmentGreen :: Color 
+pigmentGreen = "#40AA5C" 
+
+almond :: Color 
+almond = "#F0DCCA"
+
+frenchSkyBlue :: Color 
+frenchSkyBlue = "#80B2FF"
+
+blackLessTrans :: Color
+blackLessTrans = "#99000000"
+
+transparent :: Color
+transparent = "#00FFFFFF"
+
+
+black800 :: Color
+black800 = getValueFromConfig "black800"
+
+black900 :: Color
+black900 = getValueFromConfig "black900"
+
+red :: Color
+red = getValueFromConfig "red"
+
+blueMagenta :: Color 
+blueMagenta = "#9747FF"
+
+darkDescriptionText :: Color
```

Comment:
It's better to use color name, which was presented in figma design, If there's none please ask design folks to update it. So that names should not be restricted to a component

Reply:
we have 5 stars this is array, each stars have pills so another array, every pills should present in different row, for each row we have another array. so 3D Array needed. added code for reference.

```
customerFeedbackPillData :: LazyCheck -> Array (Array (Array RatingCard.FeedbackItem)) 
customerFeedbackPillData lazyCheck = [feedbackPillDataWithRating1 Language, feedbackPillDataWithRating2 Language, feedbackPillDataWithRating3 Language, feedbackPillDataWithRating4 Language, feedbackPillDataWithRating5 Language]

feedbackPillDataWithRating1 :: LazyCheck -> Array (Array RatingCard.FeedbackItem)
feedbackPillDataWithRating1 lazycheck = [
  [{id : "6", text : getString RUDE_DRIVER},
  {id : "1", text : getString FELT_UNSAFE},
  {id : "1", text : getString TOO_MANY_CALLS}],
  [{id : "6", text : getString RECKLESS_DRIVING},
  {id : "6", text : getString DRIVER_CHARGED_MORE}],
  [{id : "1", text : getString LATE_DROP_OFF},
  {id : "1", text : getString LATE_PICK_UP}]
]
```